lib/graph3d/Graph3d.js

```diff
@@ -322,18 +327,13 @@ Graph3d.prototype.getDistinctValues = function(data, column) {
 
 
 Graph3d.prototype.getColumnRange = function(data,column) {
-  var minMax;
+  var minMax  = new Range();
 
   for (var i = 0; i < data.length; i++) {
     var item = data[i][column];
-
-    if (i === 0) {
-       minMax = { min: item, max: item};
-    } else {
-      if (minMax.min > item) { minMax.min = item; }
-      if (minMax.max < item) { minMax.max = item; }
-    }
+    minMax.add(item);
```

Comment:
The idea of the call is to adjust the range so that all values passed fit in that range. Note that it's in a loop over all data points.

`init()` implies it's a one-time thing; but it's actually an adjustment of the range. So, perhaps `adjust()`?